src/main/java/org/micromanager/acqj/main/AcqEngMetadata.java

```diff
@@ -131,6 +131,9 @@ public static void addImageMetadata(JSONObject tags, AcquisitionEvent event,
          }
          if (event.getZPosition() != null) {
             AcqEngMetadata.setStageZIntended(tags, event.getZPosition());
+            // even though we are not sure, adding the intended Z position is better than not
+            // adding anything:
+            AcqEngMetadata.setZPositionUm(tags, event.getZPosition());
          }

```

Comment:
I think this is confusing. This isn't really the z position, it is only intended. So why misrepresent it as correct